system_setup/server/controllers/wslconfadvanced.py

```diff
@@ -89,20 +62,20 @@ def __init__(self, app):
             wslconfig = configparser.ConfigParser()
             wslconfig.read('/etc/wsl.conf')
```

Comment:
There is one logic issue here: what happens if wsl.conf (or ubuntu-wsl.conf) are not here? We should load some defaults settings that are set in the model.

Reply:
it already check if it exists:

```
        if path.exists('/etc/wsl.conf'):
            wslconfig = configparser.ConfigParser()
            wslconfig.read('/etc/wsl.conf')
```